xonsh/cli_utils.py

```diff
@@ -85,7 +85,21 @@ def _get_param_doc(doc: str, param: str) -> tp.Iterator[str]:
         yield lin
 
 
-def get_doc(func: tp.Union[tp.Callable, str], parameter: str = None):
+def _get_epilog_doc(doc: str) -> str:
+    lines = doc.splitlines()
+    for idx, lin in enumerate(lines):
+        if (
+            lin.startswith("-")
```

Comment:
`lin.lstrip`?

Reply:
since we use cleandoc, any padding will get removed.